packages/flutter_test/lib/src/binding.dart

```diff
@@ -62,6 +62,11 @@ enum EnginePhase {
   sendSemanticsUpdate,
 }
 
+/// Signature of callbacks to intercept messages on a given channel.
```

Comment:
nit: "...callbacks used to intercept..."

Reply:
Done!